cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1077,8 +1077,13 @@
     </property>
 
     <property>
+      <name>security.auth.server.address</name>
+      <value>asdf</value>
```

Comment:
Can we make a note that this is deprecated? We should also give devops headsup on this change.